app/uk/gov/hmrc/residencenilratebandcalculator/views/threshold_calculation_result.scala.html

```diff
@@ -18,8 +18,8 @@ <h1 class="heading-large">@messages("threshold_calculation_result.info.zero.head
         <p>@messages("threshold_calculation_result.info.zero.threshold_change")</p>
     } else {
         <h1 class="heading-large">@messages("threshold_calculation_result.info.non_zero.header")</h1>
-        <p class="heading-medium bold">@messages("threshold_calculation_result.info.non_zero.guidance")
-            <span class="data-item bold-xxlarge">@residenceNilRateAmount</span>
```

Comment:
The "data-item" class is used by the ATs to identify the location of the RNRB amount - either it will need to be added back in or test step will need to be updated @muninderadavelly @paingicool